Unexpected schema domains in dcp46 & dcp49 snapshots

After a targeted reindex of the `dcp46` catalog, the following snapshots failed to index:

**CloudWatch Logs Insights**    
region: us-east-1    
log-group-names: /aws/lambda/azul-indexer-prod-contribute, /aws/lambda/azul-indexer-prod-contribute_retry    
start-time: -50400s    
end-time: 0s    
query-string:
  ```
  fields @timestamp, @message
| filter @message like /Worker failed/
| stats count(*) as bundles by notification.bundle_fqid.source.spec as snapshot, catalog
| limit 3
  ```
---
| snapshot | catalog | bundles |
| --- | --- | --- |
| tdr:bigquery:gcp:datarepo-4e5e9f9b:hca_prod_2043c65a1cf84828a6569e247d4e64f1__20220111_dcp2_20220120_dcp12:/0 | dcp46 | 15189 |
| tdr:bigquery:gcp:datarepo-2be1e691:hca_prod_0777b9ef91f3468b9deadb477437aa1a__20220330_dcp2_20230314_dcp25:/0 | dcp46 | 1 |
| tdr:bigquery:gcp:datarepo-ff62a88c:hca_prod_7b393e4d65bc4c03b402aae769299329__20220519_dcp2_20230314_dcp25:/0 | dcp46 | 1 |
---

The indexer failed with the following AssertionError:

```
Traceback (most recent call last):
  File "/var/task/azul/indexer/index_controller.py", line 175, in contribute
    contributions, replicas = self.transform(catalog, notification, delete)
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/task/azul/indexer/index_controller.py", line 226, in transform
    results = service.transform(catalog, bundle, partition, delete=delete)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/task/azul/indexer/index_service.py", line 288, in transform
    transformers = plugin.transformers(bundle, delete=delete)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/task/azul/plugins/metadata/hca/__init__.py", line 76, in transformers
    api_bundle = api.Bundle(uuid=bundle.uuid,
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/task/humancellatlas/data/metadata/api.py", line 968, in __init__
    self.projects = from_json_vx(Project)
                    ^^^^^^^^^^^^^^^^^^^^^
  File "/var/task/humancellatlas/data/metadata/api.py", line 966, in from_json_vx
    return {entity.document_id: entity for entity in entities}
                                                     ^^^^^^^^
  File "/var/task/humancellatlas/data/metadata/api.py", line 963, in <genexpr>
    core_cls.from_json(**args)
  File "/var/task/humancellatlas/data/metadata/api.py", line 121, in from_json
    described_by = cls.validate_described_by(content)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/var/task/humancellatlas/data/metadata/api.py", line 132, in validate_described_by
    assert described_by.netloc in valid_schema_domains, R(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: R('Unexpected schema domain', furl('https://schema.dev.data.humancellatlas.org/type/project/14.0.0/project'), ['schema.humancellatlas.org'])
```

--- a/deployments/prod/environment.py
+++ b/deployments/prod/environment.py
@@ -1342,7 +1342,7 @@ dcp46_sources = mkdict(dcp45_sources, 489, mkdelta([
     # @formatter:on
 ]))
 
-dcp47_sources = mkdict(dcp46_sources, 495, mkdelta([
+dcp47_sources = mkdict(dcp46_sources, 494, mkdelta([
     mksrc('bigquery', 'datarepo-3c5378af', 'hca_prod_03c6fce7789e4e78a27a664d562bb738__20220110_dcp2_20250304_dcp47'),
     mksrc('bigquery', 'datarepo-2590222b', 'hca_prod_04ad400c58cb40a5bc2b2279e13a910b__20220114_dcp2_20250304_dcp47'),
     mksrc('bigquery', 'datarepo-f35c4c99', 'hca_prod_04e4292cf62f4098ae9bfd69ae002a90__20230427_dcp2_20250304_dcp47'),
@@ -1379,7 +1379,6 @@ dcp47_sources = mkdict(dcp46_sources, 495, mkdelta([
     mksrc('bigquery', 'datarepo-80477604', 'hca_prod_1ebe8c34454e4c28bd713a3e8b127be4__20221208_dcp2_20250304_dcp47'),
     mksrc('bigquery', 'datarepo-40766c2f', 'hca_prod_1fa8b11f56fa45a6a7776af70e17a6b3__20220928_dcp2_20250304_dcp47'),
     mksrc('bigquery', 'datarepo-fa586d18', 'hca_prod_1fac187b1c3f41c4b6b66a9a8c0489d1__20230427_dcp2_20250304_dcp47'),
-    mksrc('bigquery', 'datarepo-749cccac', 'hca_prod_2043c65a1cf84828a6569e247d4e64f1__20220111_dcp2_20250304_dcp47'),
     mksrc('bigquery', 'datarepo-1ffce377', 'hca_prod_2084526ba66f4c40bb896fd162f2eb38__20220111_dcp2_20250304_dcp47'),
     mksrc('bigquery', 'datarepo-05637602', 'hca_prod_2086eb0510b9432bb7f0169ccc49d270__20220111_dcp2_20250304_dcp47'),
     mksrc('bigquery', 'datarepo-31f6c2b1', 'hca_prod_20f37aafcaa140e69123be6ce8feb2d6__20220111_dcp2_20250304_dcp47'),

